bc_obps/registration/api/user_operator.py

```diff
@@ -181,11 +179,7 @@ def get_user_operator_operator_id(request):
 @authorize(AppRole.get_all_authorized_app_roles(), UserOperator.get_all_industry_user_operator_roles())
 def get_user_operator(request, user_operator_id: int):
     user: User = request.current_user
-    user_operator = get_object_or_404(UserOperator, id=user_operator_id)
-    if user.is_industry_user():
-        authorized_users = get_an_operators_approved_users(user_operator.operator)
-        if user.user_guid not in authorized_users:
-            raise HttpError(401, UNAUTHORIZED_MESSAGE)
+    user_operator = get_object_or_404(UserOperator, id=user_operator_id, user=user.user_guid)
```

Comment:
Allow users to get any user operator that they created instead of only approved ones. This is necessary to allow them to view forms they have submitted but aren't approved yet.